src/test/java/seedu/address/model/room/RoomIntegrationTest.java

```diff
@@ -25,7 +25,9 @@ public void addTask_nullTask_throwsNullPointerException() {
     public void addTask_validTask_success() {
         Room room = new RoomBuilder(ROOM_PATIENT_ALICE_NO_TASK).build();
         room.addTask(REMIND_PATIENT);
-
+        System.out.println(room.getTaskList().getInternalList().size());
```

Comment:
remember to remove the system.out.print after doing the "actual" testing

Reply:
Okay, thanks for noticing.